src/core/ipfs/init.js

```diff
@@ -85,7 +85,7 @@ module.exports = function init (self) {
           throw err
         }
         const index = __dirname.lastIndexOf('/')
-        async.eachLimit(res, 10, (element, callback) => {
+        parallelLimit(res.map((element) => (callback) => {
```

Comment:
I wonder if we should start calling `next` the functions that are not exactly `callbacks` in the async way.